pydash/numerical.py

```diff
@@ -383,6 +385,52 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, op):
```

Comment:
The default value should be an argument to this function that the main math function will pass in.